app/src/ui/changes/no-changes.tsx

```diff
@@ -657,17 +670,71 @@ export class NoChanges extends React.Component<
     const title = `Create a Pull Request from your current branch`
     const buttonText = `Create Pull Request`
 
+    if (!enableStartingPullRequests()) {
+      return (
+        <MenuBackedSuggestedAction
+          key="create-pr-action"
+          title={title}
+          menuItemId={'create-pull-request'}
+          description={description}
+          buttonText={buttonText}
+          discoverabilityContent={this.renderDiscoverabilityElements(
+            createMenuItem
+          )}
+          type="primary"
+          disabled={!createMenuItem.enabled}
+          onClick={this.onCreatePullRequestClicked}
+        />
+      )
+    }
+
+    const startMenuItem = this.getMenuItemInfo('start-pull-request')
+
+    if (startMenuItem === undefined) {
+      log.error(`Could not find matching menu item for 'start-pull-request'`)
+      return null
+    }
+
+    const createPullRequestAction: IDropdownSuggestedActionOption = {
+      title,
+      label: buttonText,
+      description,
+      value: PullRequestSuggestedNextAction.CreatePullRequest,
+      menuItemId: 'create-pull-request',
+      discoverabilityContent:
+        this.renderDiscoverabilityElements(createMenuItem),
+      disabled: !createMenuItem.enabled,
+      onClick: this.onCreatePullRequestClicked,
+    }
+
+    const previewPullRequestAction: IDropdownSuggestedActionOption = {
+      title: `Preview the Pull Request from your current branch`,
+      label: 'Preview Pull Request',
+      description: (
+        <>
+          The current branch (<Ref>{tip.branch.name}</Ref>) is already published
+          to GitHub. Preview the changes this pull request will have before
+          proposing your changes.
+        </>
+      ),
+      value: PullRequestSuggestedNextAction.PreviewPullRequest,
+      menuItemId: 'start-pull-request',
+      discoverabilityContent: this.renderDiscoverabilityElements(startMenuItem),
+      disabled: !createMenuItem.enabled,
+    }
+
+    const pullRequestActions: ReadonlyArray<IDropdownSuggestedActionOption> = [
+      previewPullRequestAction,
+      createPullRequestAction as IDropdownSuggestedActionOption,
```

Comment:
I think this cast is not needed? 🤔 
```suggestion
      createPullRequestAction,
```

(You might require to run the linter if you commit this change)